README.rst

```diff
@@ -3,10 +3,9 @@ PyRocky
 
 |pyansys| |MIT| |python| |pypi| |codecov| |MIT| |black| |pre-commit|
 
-PyRocky is a Python client library for remotely controlling
-`Ansys Rocky <https://www.ansys.com/products/fluids/ansys-rocky>`_,
-which is the most powerful software in the market for performing
-DEM (discrete element method) simulations.
+PyRocky is a Python client library to create, run and post-processing
```

Comment:
```suggestion
PyRocky is a Python client library to create, run, and postprocess
```
Oxford comma and simple verb form. (An Oxford comma refers to always putting a comma before an "and" or an "or" in a list of three or more items.